lib/blockscore/errors/invalid_request_error.rb

```diff
@@ -15,15 +15,17 @@ class InvalidRequestError < APIError
     # end
     def initialize(response)
       super
-      @param = @http_body[:error][:param]
+      @param = error_advice[:param]
```

Comment:
In this case I do NOT believe param is always present. That is just a strong inference.